virttest/tests/unattended_install.py

```diff
@@ -1129,7 +1130,14 @@ def copy_images():
                 logging.warn(e)
             from virttest import utils_test
             error_context.context("Copy image from NFS Server")
+            image = params.get("images").split()[0]
+            t_params = params.object_params(image)
+            qemu_image = qemu_storage.QemuImg(t_params, data_dir.get_data_dir(), image)
+            ver_to = utils_test.get_image_version(qemu_image)
             utils_test.run_image_copy(test, params, env)
+            qemu_image = qemu_storage.QemuImg(t_params, data_dir.get_data_dir(), image)
+            ver_from = utils_test.get_image_version(qemu_image)
```

Comment:
Aefore utils_test.run_image_copy(), the image is created by command qemu-img on the test host.
After the function, the image is from nfs server.